### ISSUE TITLE
want to disable auto-connect

### ISSUE THREAD

**Issue by satyadeepk** [Jigsaw-Code/outline-client#492]
Hi, 
Thanks for the super-easy VPN setup tool.
I am wondering if there is a way to turn off the auto-connect feature on system start in Mac. We are planning to use Outline to connect to our private services on AWS and we don't always want to start VPN by default.

If not, it will be a great little feature addition in Outline client settings page.

**Comment by trevj** [comment]
Possibly related:
https://github.com/Jigsaw-Code/outline-client/issues/183

**Comment by sarunint** [comment]
@trevj I'm seeing a different behaviour on Windows. I disconnect from the server, but on the system tray the icon still indicates that I'm still connected. I checked and I'm really disconnected. When I restart my computer, Outline tried to reconnect on startup every time regardless of whether I disconnect or not.

**Comment by trevj** [comment]
@sarunint Correct, the issue of the system tray not updating on disconnection is a bug that was introduced recently. I submitted a fix a couple of days ago and hope to release it later today:
https://github.com/Jigsaw-Code/outline-client/pull/506

**Comment by HazyFish** [comment]
It may be better to provide an option in the app for users to turn off the feature, because some users are just not aware that the device is connected to Outline server at boot, although there is already a notice telling the user about it after installation.

**Comment by trevj** [comment]
Folks, apologies for the slow reply. I'm going to add this as a feature request - we'll have to think about how this will work across platforms.
